gef-ui/component-tests/partials/application-activity.component-test.js

```diff
@@ -40,7 +36,19 @@ describe(page, () => {
     wrapper.expectElement('[data-cy="application-banner"]').toExist();
   });
 
-  it('should render timeline', () => {
+  it('should render a timeline', () => {
     wrapper.expectElement('[data-cy="portal-activities-timeline"]').toExist();
   });
+
+  it('should render an activity title', () => {
+    wrapper.expectText(`[data-cy="activity-${mockActivity.title}-title"]`).toRead(mockActivity.title);
+  });
+
+  it('should render an activity byline', () => {
+    wrapper.expectText(`[data-cy="activity-${mockActivity.title}-byline"]`).toRead(`by ${mockActivity.byline}`);
+  });
+
+  it('should render an activity date', () => {
+    wrapper.expectText(`[data-cy="activity-${mockActivity.title}-time"]`).toRead(`${mockActivity.date} ${mockActivity.time}`);
+  });
```

Comment:
can you check this renders the different activity types correctly

Reply:
e.g. check for `datacy="previous-status-tag"`

Reply:
Have manually checked activities AFAIK - all seems to be ok. Fixed an issue picked up in E2E tests. Screenshot of the latest:

<img width="334" alt="Screenshot 2024-12-10 at 15 06 18" src="https://github.com/user-attachments/assets/754b375e-8c3c-46d1-9d09-5e55c660054a">

Reply:
could you check if the previous status and the new status tag exist in this test?

Reply:
Sorry chaps, forgot to update component tests. Have updated.

Reply:
Marking as resolved for merge